### PR TITLE
Release for July 31st, 2023

### PR DESCRIPTION
**Pull request by artpark**
### Summary

**What was changed:**

**All minor** version releases:
  - terra-abstract-modal@3.43.0
  - terra-aggregator@4.75.0
  - terra-application-header-layout@3.39.0
  - terra-application-layout@5.73.0
  - terra-application-links@6.69.0
  - terra-application-menu-layout@3.32.0
  - terra-application-name@3.43.0
  - terra-application-navigation@1.70.0
  - terra-application-utility@2.52.0
  - terra-brand-footer@3.11.0
  - terra-collapsible-menu-view@6.75.0
  - terra-date-input@1.42.0
  - terra-date-picker@4.86.0
  - terra-date-time-picker@4.91.0
  - terra-dialog-modal@3.88.0
  - terra-embedded-content-consumer@3.37.0
  - terra-form-validation@1.83.0
  - @cerner/terra-framework-docs@1.28.0
  - terra-hookshot@5.41.0
  - terra-layout@4.38.0
  - terra-menu@6.73.0
  - terra-modal-manager@6.66.0
  - terra-navigation-layout@5.39.0
  - terra-navigation-prompt@1.74.0
  - terra-navigation-side-menu@2.47.0
  - terra-notification-dialog@4.33.0
  - @cerner/terra-pills@1.10.0
  - terra-popup@6.70.0
  - terra-slide-group@4.32.0
  - terra-slide-panel-manager@5.75.0
  - terra-slide-panel@3.44.0
  - terra-tabs@7.1.0
  - terra-theme-provider@4.13.0
  - terra-time-input@4.54.0

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [x] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-XXXX 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-worklist-data-grid/CHANGELOG.md
Comment (sdadn) [inline packages/terra-worklist-data-grid/CHANGELOG.md:1]: Can we hold off on releasing worklist-data-grid right now? I would like to merge in https://github.com/cerner/terra-framework/pull/1719 first as it addresses an issue with pinned columns.

File: packages/terra-framework-docs/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 1.28.0 - (July 31, 2023)
+
+* Changed
+  * Minor dependency version bump
+
 ## 1.26.0 - (July 20, 2023)
```
Comment (sdadn) [inline packages/terra-framework-docs/CHANGELOG.md:10]: Can you remove this line? Sorry this got added from a merge conflict by https://github.com/cerner/terra-framework/pull/1710.

File: packages/terra-worklist-data-grid/CHANGELOG.md
Comment (artpark) [reply packages/terra-worklist-data-grid/CHANGELOG.md:1]: [3b02a86](https://github.com/cerner/terra-framework/pull/1720/commits/3b02a863852655bc86a0cdd9a52a0801f17b27b5) Removed

File: packages/terra-framework-docs/CHANGELOG.md (hunk 1 above)
Comment (artpark) [reply packages/terra-framework-docs/CHANGELOG.md]: [3b02a86](https://github.com/cerner/terra-framework/pull/1720/commits/3b02a863852655bc86a0cdd9a52a0801f17b27b5) Removed

**Review by sdadn — APPROVED** [review]

**Review by trandrew1023 — APPROVED** [review]

**Pull request merged**